src/derivation.test.ts

```diff
@@ -235,61 +249,36 @@ describe('derivation', () => {
         await expect(
           bip32Derive({
             path: input,
-            privateKey: Buffer.alloc(32).fill(1),
-            chainCode: Buffer.alloc(32).fill(1),
+            node,
           }),
         ).rejects.toThrow(
           'Invalid BIP-32 index: The index must be a non-negative decimal integer less than 2147483648.',
         );
       }
+    });
 
+    it('throws when no node is specified', async () => {
       await expect(
         bip32Derive({
-          path: `44`,
-          chainCode: Buffer.alloc(32).fill(1),
+          path: '0',
         }),
       ).rejects.toThrow(
-        'Invalid parameters: Must specify either a parent private or public key.',
+        'Invalid parameters: Must specify a node to derive from.',
       );
     });
 
-    it('throws for an invalid private key', async () => {
```

Comment:
Why are we removing these tests?

Reply:
The function doesn't accept a plain private key anymore, and `SLIP10Node` validates any inputs upon creating the node.